### DISCUSSION TITLE
Other architecture support?

### DISCUSSION THREAD

**Discussion by ghost** [turboderp/exllamav2#132] [Ideas]
Would it be possible to ever implement the support for other architectures like OPT for this? You can only run these models using very inefficient loaders right now.

**Comment by turboderp** [comment]
It would certainly be possible, yes. It's just a lot of work, and it's a commitment too, so I'm not sure exactly where it is on the roadmap given so many other priorities right now. It would definitely be a very select set of architectures, though. Would need to first of all research which architectures are most wanted, and then pick one or two among those that might actually stick around for more than a few weeks.

**Reply by ghost** [reply]
I'm mostly interested in OPT (and Neo-X) because of all the old models, but I guess that won't be done since they're already deprecated. Did you add in Mistral support? Seemed to work last time I tried.
